ui/app/models/config-ui/message.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Model, { attr } from '@ember-data/model';
+import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
+
+export default class MessageModel extends Model {
+  @attr('string') id; // 32 hex uuid generated by the backend
```

Comment:
Pretty sure `id` is a built-in model attribute so we don't need to add it here manually

Reply:
Good to know - I'll remove it!